eng/testing/AppleRunnerTemplate.sh

```diff
@@ -39,14 +35,17 @@ while true; do
     fi
 done
 
-XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
+XCODE_PATH="`xcode-select -p`/../.."
+export XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
 
-dotnet xharness ios test --app="$APP_BUNDLE" \
-    --targets=$TARGET \
+dotnet xharness ios test  \
```

Comment:
Will we support filtering tests or any xunit parameters? Like `-notrait Category=Outerloop`? If so, this is wrong and should use `RunScriptCommand` in the MSBuild side to set this command.

Reply:
Ideally yes, but we first need a way to flow arbitrary command line arguments to the iOS/Android test apps and that will need some changes in xharness and the iOS/Android test runners, I don't think we need to block the PR on that.

Reply:
Sounds good. We just need to be conscious that PR validation is running outerloop tests for iOS as well, which might be causing the slow runs.

Reply:
ah no, we added handling for that already with https://github.com/dotnet/runtime/blob/3c1b48ee29f13cac93a86829374b917375a49a2d/eng/testing/tests.mobile.targets#L76 which is read by the testrunner so Outerloop and failing tests should already be excluded.

Reply:
Ok, sounds good. Maybe we should do it how we do it in other places passing them as an argument but that is in the future, no need to block this PR.